### PR TITLE
feat: add service-map

### PR DESCRIPTION
**Pull request by Guiso92**

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -66,6 +80,10 @@ const Home: NextPage<HomeProps> = ({
       locales={LOCALES}
       strings={strings}
       menuOverlayItems={menuOverlayItems}
+      headerBannerProps={{
+        ...headerBannerStrings,
+        socialMediaData: getSocialMediaProps(socialMediaLinks),
```
Comment (AnnKats93) [inline pages/index.tsx:85]: You need to invoke getSocialMediaProps(dynamicContent) in getStaticProps() and pass them as part of headerBannerProps directly to this comopnent

File: pages/index.tsx [hunk 2]
```diff
@@ -113,12 +131,34 @@ export const getStaticProps: GetStaticProps = async ({ locale }) => {
...
     props: {
       currentLocale,
       strings,
       menuOverlayItems,
+      serviceMapProps: {
+        regions,
+        serviceCategories,
+        services,
+        defaultCoords: MAP_DEFAULT_COORDS,
+        shareButton,
```
Comment (AnnKats93) [inline pages/index.tsx:160]: Nit: you can just call getShareButtonProps(dynamicContent) here directly

File: pages/index.tsx [hunk 3]
```diff
@@ -43,9 +53,11 @@ import {
 interface HomeProps {
   currentLocale: Locale;
   strings: HomePageStrings;
+  headerBannerStrings: HeaderBannerStrings;
```
Comment (AnnKats93) [inline pages/index.tsx:56]: I think you can just pass headerBannerProps and generate them in getStaticProps.
Also, remove socialMediaLinks from here.

**Review by AnnKats93 — COMMENTED** [review]
Looks good!
Could you test Generic Template and see if Service map renders correctly? And add screenshot here.
To do that, use 'https://signpost-global.zendesk.com' in url.ts / getZendeskUrl() method and use real zendesk import instead of zendesk-fake in Home page.

**Comment by Guiso92** [comment]
image
 the tiles are not visible because I didn't create an .env to pass the mapbox token needed to retrieve them, but is fetching the data correctly from the cms

**Review by AnnKats93 — APPROVED** [review]

**Pull request merged**